src/modules/games/scoresaber/gains.ts

```diff
@@ -12,7 +12,7 @@ export default class Gains extends SubCommand {
   constructor(parent: Command | SubCommand) {
     super(parent, {
       options: {
-        aliases: ["losses"],
+        aliases: ["losses"], ["sustain"],
```

Comment:
This will cause a compilation error, the alias should be appended in the array as so:
```
aliases: ["losses", "sustain"]
```